src/common.js

```diff
@@ -57,6 +67,19 @@ async function updatePackageJSON(serveInstalled, buildPath, packageJSON, account
             }
         }
     ]
+    if (buildPath) {
+        availablePatches.push({
+            op: 'add',
+            path: '/scripts/start',
+            value: `serve -s ${buildPath} -l 8080`
+        })
+    } else {
+        availablePatches.push({
+            op: 'add',
+            path: '/scripts/start',
+            value: 'node YOUR_ENTRYPOINT.js'
```

Comment:
this would deploy with a validation error. instead perhaps just check if buildpath is undefined before anything else is run, and then just show usage of the static command